client/signup/controller.js

```diff
@@ -226,6 +227,19 @@ export default {
 
 		store.set( 'signup-locale', localeFromParams );
 
+		const isOnboardingFlow = flowName === 'onboarding';
+		const lang = localeFromParams ?? localeFromStore;
+		if ( isOnboardingFlow && lang && lang.toLowerCase() !== 'en' ) {
```

Comment:
Do you need the `lang` check? Won't this be targeted by ExPlat/assignment?

Reply:
Good catch, probably not. I'll get back to this

Reply:
Removed the check!